main.js

```diff
@@ -0,0 +1,92 @@
+let books = [
+  {
+    title: 'The Tiral',
+    author: 'Franz Kafka',
+  },
+  {
+    title: 'Harry Potter',
+    author: 'J. K. Rowling',
+  },
+];
+
+// add data to local storage
+
+function addToLocalStorage(data) {
+  localStorage.setItem('data', JSON.stringify(data));
+}
+
+// Remove book from the list
+
+function removeBook(ref) {
+  const result = books.filter((value) => value.title !== ref);
+  books = result;
```

Comment:
- [ ] There seems to be a bug in your app. As demonstrated in the following images, when I add duplicate books and remove one, the right book is removed on the webpage, however, every instance of the book on the `localStorage` is also removed. This makes it such that whenever I reload the page, all books of the same title are removed from the page (because `localStorage` is now empty). Kindly resolve this.

  **SUGGESTION:** Here, instead of removing books using the `title` attribute, I would suggest you implement your deletion using some kind of **unique** `id`. This `id` attribute can be added upon book creation.

  Duplicates added | One Book is removed | Page is reloaded
  :---:|:---:|:---:
  ![awais-1st](https://user-images.githubusercontent.com/36542986/152881416-d216679b-3662-4938-8f56-b4f2106c51ae.jpg) | ![awais-2nd](https://user-images.githubusercontent.com/36542986/152881803-e209c7e5-a841-485b-9161-38d6d461aa9a.jpg) | ![awais-3rd](https://user-images.githubusercontent.com/36542986/152881422-07ab4448-907c-4010-b57b-ef593677d0be.jpg)

Reply:
Thanks for pointing out this bug. I have changed the removeBook function so that now it's using a unique `id` for each instance of the book instead of a `title`